login/login-render-engine-impl/impl/src/webapp/vm/defaultskin/xlogin.vm

```diff
@@ -74,6 +75,16 @@
 
             //Add an on-load listener to focus the password field
             window.addEventListener("load", () => document.querySelector("#eid")?.focus());
+
+            // Remove the the error class from the form fields when the user starts typing
+            const formFields = document.querySelectorAll(".form-control");
+            const removeInvalid = el => formFields.forEach(el => el.classList.remove("is-invalid"));
+            formFields.forEach(input => {
+            
+                input.addEventListener("input", removeInvalid);
+                input.addEventListener("click", removeInvalid);
```

Comment:
Maybe the once option on the eventlistener would make sense here, what do you think?
`input.addEventListener("click", removeInvalid, { once: true });`
I think this classes are not re-added?